cypher/cypher-docs/src/docs/dev/deprecations-additions-and-compatibility.asciidoc

```diff
@@ -745,6 +745,19 @@ New columns have been added to the result:
 * `lastStopTime`
 
 These columns are only returned in the full result set (with `YIELD`) and not by default.
+
+a|
+label:functionality[]
+label:updated[]
+[source, cypher, role="noheader"]
+----
+RETURN round(val, precision)
+----
+a|
+Rounding infinity and NaN values will newly return the original value instead of
+returning an integer approximation for precision 0 (Inf -> 9223372036854776000.0, Nan -> 0) or
+throwing an exception for precision > 0.
```

Comment:
This was kind of confusing, I think providing an example that shows that rounding infinity will be infinity would be easier to see. My first read thought that it would return what was shown in the parenthesis 🙈

Also just a little nitpick, I think using the word "newly" is a bit strange, and would instead say "now".